extensions/clickup/src/views/DocList/ListDocs.tsx

```diff
@@ -0,0 +1,30 @@
+import { Action, ActionPanel, Icon, List } from "@raycast/api";
+import useDocs from "../../hooks/useDocs";
+import ListDocPages from "./ListDocPages";
+
+export default function ListDocs({ workspaceId, workspaceName }: { workspaceId: string; workspaceName: string }) {
```

Comment:
Does this need to be default exported? Other functions have not been

Reply:
Muscle memory 🫠.
Fixed.